Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(JSON: Dictionary<String, AnyObject>) {
```

Comment:
I'll have to think this over a bit. I'd love to get some feedback from @thoughtbot/ios and/or @hyperspacemark on this as well.